opty/direct_collocation.py

```diff
@@ -188,7 +203,30 @@ def __init__(self, equations_of_motion, state_symbols,
             to a directory here.
 
         """
-        self.eom = equations_of_motion
+        Lambda_Matrix = sym.Matrix([[0,0,0,0], [0,0,0,0], [0,0,Lambda,0],[0,0,0,Lambda]])
```

Comment:
Why is this of fixed size? Shouldn't it be a function of the number of states, number of inputs, etc?